server/container_status.go

```diff
@@ -69,3 +78,33 @@ func (s *Server) ContainerStatus(ctx context.Context, req *pb.ContainerStatusReq
 	logrus.Debugf("ContainerStatusResponse: %+v", resp)
 	return resp, nil
 }
+
+func (s *Server) getMounts(id string) ([]*pb.Mount, error) {
+	config, err := s.store.GetFromContainerDirectory(id, "config.json")
+	if err != nil {
+		return nil, err
+	}
+	var m rspec.Spec
+	if err = json.Unmarshal(config, &m); err != nil {
+		return nil, err
+	}
+	mounts := []*pb.Mount{}
+	for _, b := range m.Mounts {
+		if b.Type != "bind" {
```

Comment:
@cyphar Good point. Even though we set Type to bind in runc but we should go by the spec. https://github.com/opencontainers/runc/blob/master/libcontainer/specconv/spec_linux.go#L253